riscv/src/runtime.rs

```diff
@@ -148,6 +148,29 @@ impl Runtime {
         self.syscalls.contains_key(&s)
     }
 
+    pub fn with_keccak(mut self) -> Self {
+        self.add_submachine(
+            "std::machines::hash::keccakf::KeccakF",
+            None,
+            "keccakf",
+            [format!(
+                "instr keccakf ~ keccakf.keccakf {};", // operation
+                instr_register_params(0, 2, 2)         // 2 pointers as both input and output
+            )],
+            0,
+            std::iter::once("keccakf;".to_string()) // must be called at least once
+                .chain((0..2).map(|i| format!("{} <=X= 0;", reg(i)))), // zero out output registers
```

Comment:
I think this code here needs to do the following (for now, as it won't work with continuations, per @georgwiese 's comment):
- point the first two registers to distinct memory addresses
- call `keccakf`
- zero out memory pointed out by second register (i.e., output written by keccakf)

Reply:
What do you mean pointing the first two registers to distinct memory addresses? Do you mean randomized or I thought they will be specified by input arguments `x10` and `x11`?